### ISSUE TITLE
So do we still need Browsersync even with Webpack's HMR?

### ISSUE THREAD

**Issue by jitendravyas** [bionikspoon/webpack-hmr-wordpress#3]

**Comment by manuphatak** [comment]
Yes and No.  BrowserSync and HMR solves 2 different problem.  HMR does the hot reloads.  BrowserSync lets you open multiple browsers and sync actions (clicking, scrolling, navigating, etc) while the page is open.  Basically, freaking awesome for building responsive sites.  To answer your question, you don't need to use BrowserSync.
